src/main/java/Deadline.java

```diff
@@ -0,0 +1,37 @@
+public class Deadline extends Task {
+
+    private String time;
+
+    /**
+     *  This method updates the action and time accordingly for Deadline.
+     * @param description This is the action to be taken.
+     * @param time This is the time for which that action has to happen.
+     */
+    public Deadline (String description, String time){
+        super(description);
+        this.time = time;
+    }
+
+    @Override
+    /*Override the toString method for Deadline*/
+    public String toString(){
+        return "[D]" + super.toString() + "| by: " + time ;
+    }
+
+    /**
+     * This method indicates that a new Deadline Task has been added to the ArrayList in Store.
+     */
+    public void Output(){
```

Comment:
You might want to consider changing your method name adhering to the style guide.